src/templates/docPage.js

```diff
@@ -179,28 +182,61 @@ const BasicDoc = ({ data, location, pageContext }) => {
           </Layout.Content>
         </LoggedInProvider>
         {!hideNavs && (
-          <Layout.PageTools
-            css={css`
-              @media screen and (max-width: 1240px) {
-                margin-top: 1rem;
-                position: static;
-              }
-            `}
+          // TODO pass nodeRef to avoid `findDOMNode` usage
```

Comment:
this todo isn't _strictly_ necessary, it's just recommended. passing a `ref` through to PageTools (which is in the theme) would just be 😩 so i'm inclined to say it's ok for now